mmv1/products/dialogflowcx/terraform.yaml

```diff
@@ -105,6 +105,29 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       displayName: !ruby/object:Overrides::Terraform::PropertyOverride
         validation: !ruby/object:Provider::Terraform::Validation
           function: 'validation.StringLenBetween(0, 64)'
+  Environment: !ruby/object:Overrides::Terraform::ResourceOverride
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 40
+      update_minutes: 40
+    custom_code: !ruby/object:Provider::Terraform::CustomCode
+      custom_import: templates/terraform/custom_import/dialogflowcx_environment.go.erb   
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dialogflowcx_environment_full"
+        primary_resource_id: "development"
+        vars:
+          agent_name: "dialogflowcx-agent"
+    skip_sweeper: true
+    id_format: "{{parent}}/environments/{{name}}"
```

Comment:
I don't think this will work, as `{{name}}` will be the long-form name that includes `{{parent}}/environments/`

Are you seeing the import tests work for this resource?

Reply:
Yes you are right is better with long-form way so I changed it

Tests are working

![image](https://user-images.githubusercontent.com/27365102/125851502-f0d8af66-35c6-4941-a652-e4f128023145.png)